### PR TITLE
Polymorphism

### PR DESCRIPTION
**Pull request by cerilacanda**

### CODE REVIEW

**Review by clydiex — APPROVED** [review]

File: content/Polymorphism/Polymorphism.q.md [hunk 1]
```diff
@@ -0,0 +1,1490 @@
...
+ - Access level to `Square::calculateArea()` must be public (as in class MyShape) in `Square.php` on line number 4
+
+ - Missing argument `1` for `Circle::__construct()`, called in `Main.php` on line 5 and defined in `Circle.php` on line number 9
+
+ - Class `Circle` contains `1` abstract method and must therefore be declared abstract or implement the remaining methods (MyShape::calculateArea) in `Circle.php` on line number 4
+
+
+/// type=MS, answer=[3,4,5]
+
+Which statements correctly describe the error?
```
Comment (clydiex) [inline content/Polymorphism/Polymorphism.q.md:798]: Option 4 should be placed before Option 3 for this question.

File: content/Polymorphism/Polymorphism.q.md [hunk 2]
```diff
@@ -0,0 +1,1490 @@
...
+ - Access level to `Square::calculateArea()` must be public (as in class MyShape) in `Square.php` on line number 4
+
+ - Missing argument `1` for `Circle::__construct()`, called in `Main.php` on line 5 and defined in `Circle.php` on line number 9
+
+ - Class `Circle` contains `1` abstract method and must therefore be declared abstract or implement the remaining methods (MyShape::calculateArea) in `Circle.php` on line number 4
+
+
+/// type=MS, answer=[2,5]
+
+Which statements correctly describe the error message?
```
Comment (clydiex) [inline content/Polymorphism/Polymorphism.q.md:1136]: Option 5  must be placed after option 2 for this question.

File: content/Polymorphism/Polymorphism.q.md [hunk 3]
```diff
@@ -0,0 +1,1490 @@
...
+ - Access level to `Square::calculateArea()` must be public (as in class MyShape) in `Square.php` on line number 4
+
+ - Missing argument `1` for `Circle::__construct()`, called in `Main.php` on line 5 and defined in `Circle.php` on line number 9
+
+ - Class `Circle` contains `1` abstract method and must therefore be declared abstract or implement the remaining methods (MyShape::calculateArea) in `Circle.php` on line number 4
+
+
+/// type=MS, answer=[3,4]
+
+Which statements correctly describe the error?
```
Comment (clydiex) [inline content/Polymorphism/Polymorphism.q.md:1310]: Option 3 must be placed after option 1 for this question.

**Review by clydiex — APPROVED** [review]

**Review by rosoarv — COMMENTED** [review]
We need to mention also the phrase `many forms` which importantly describes the word Polymorphism.

**Review by rosoarv — APPROVED** [review]

**Comment by cerilacanda** [comment]
> We need to mention also the phrase `many forms` which importantly describes the word Polymorphism.

I did not include the phrase `many forms` for I find it confusing but instead I replace it with `can have different implementations and behaviors for different instances`. Thank you.

**Review by gjoren4 — APPROVED** [review]
Done  reviewing. Looks good!

**Pull request merged**